x-pack/plugin/ccr/qa/downgrade-to-basic-license/build.gradle

```diff
@@ -11,7 +11,7 @@ dependencies {
 }
 
 task "leader-cluster"(type: RestIntegTestTask) {
-  mustRunAfter(precommit)
+  mustRunAfter("precommit")
```

Comment:
I feel like the rest test plugin or something should be adding this ordering rule.

Reply:
you mean that tasks of type `RestIntegTestTask` should always run after `precommit` ?

Reply:
Yeah, I would think so.